pkg/aws/client.go

```diff
@@ -1212,27 +1212,39 @@ func (c *awsClient) DeleteSecretInSecretsManager(secretArn string) error {
 }
 
 func (c *awsClient) GetSecurityGroupIds(vpcId string) ([]ec2types.SecurityGroup, error) {
```

Comment:
Would it be worth updating the naming of this function, as this returns a list of security groups not just their id's

Reply:
updated